ssg/jinja.py

```diff
@@ -45,7 +45,8 @@ def uptodate():
 def _get_jinja_environment(substitutions_dict):
     if _get_jinja_environment.env is None:
         bytecode_cache = None
-        if required_key(substitutions_dict, "jinja2_cache_enabled") == "true":
+        have_key = 'jinja2_cache_enabled' in substitutions_dict
+        if have_key and substitutions_dict["jinja2_cache_enabled"] == "true":
```

Comment:
nicer way to do this is `substitutions_dict.get("jinja2_cache_enabled") == "true"`

Reply:
@mpreisler Good point, forgot about `get`. :) Updated.